### PR TITLE
[chain] bind specific port for tunnel and add msg update route

### PR DESCRIPTION
**Pull request by taobun**
Fixed: #XXXX

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

**Review by RogerKSI — CHANGES_REQUESTED** [review]

**Review by RogerKSI — APPROVED** [review]

File: proto/band/tunnel/v1beta1/tx.proto [hunk 1]
```diff
@@ -68,11 +71,26 @@ message MsgCreateTunnelResponse {
...
 
-// MsgUpdateAndResetTunnel is the transaction message to update a tunnel information
-// and reset the interval.
-message MsgUpdateAndResetTunnel {
+// MsgUpdateRoute is the transaction message to update a route information of the tunnel.
+message MsgUpdateRoute {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name)           = "tunnel/MsgUpdateRoute";
+
+  // tunnel_id is the ID of the tunnel to edit.
```
Comment (satawatnack) [inline proto/band/tunnel/v1beta1/tx.proto:79]: to update

File: x/tunnel/keeper/ibc_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		{
+			name:      "without dot",
+			portID:    "tunnel1",
+			expResult: false,
+		},
+		{
+			name:      "invalid prefix",
+			portID:    "tun.1",
+			expResult: false,
+		},
```
Comment (satawatnack) [inline x/tunnel/keeper/ibc_test.go:35]: in this test `tunnel.eiei` should we also verify the number as well

File: x/tunnel/keeper/ibc.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+const portIDPrefix = "tunnel."
+
+func PortIDForTunnel(tunnelID uint64) string {
+	return fmt.Sprintf("%s%d", portIDPrefix, tunnelID)
+}
+
+func IsValidPortID(portID string) bool {
+	return strings.HasPrefix(portID, portIDPrefix)
+}
```
Comment (satawatnack) [inline x/tunnel/keeper/ibc.go:41]: add comment

Comment (RogerKSI) [reply x/tunnel/keeper/ibc.go:41]: added comment.

**Review by satawatnack — APPROVED** [review]

**Pull request merged**